### DISCUSSION TITLE
Issue Templates Don't Work on Mobile/Android

### DISCUSSION THREAD

**Discussion by FWC-9696** [community/community#118586] [Mobile] [answered]
When attempting to submit an issue with a template, clicking the link to the template will redirect you back to the main issue reporting screen.

In the image below, clicking "bug report" or "feature request" will both kick you back to this same screen when GitHub links are set to open in the app.

![Screenshot_20240410-123515_GitHub.png](https://github.com/community/community/assets/76880537/7c709d0d-744f-4a73-b03a-9a79c42a6db9)

**Comment by eliperkins** [comment, marked as answer]
Hey @FWC-9696 , thanks for this feedback! It seems like the repo in your screenshot uses [issue forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms), which are currently in beta and not supported by GitHub Mobile. I can understand how this can be frustrating, and I've added your notes to an internal tracking issue to get issue forms on mobile supported. 🙇 

Until then, are you unblocked to submit issues from the link out to the web browser from GitHub Mobile? If not, I'd love to dig in more!

Thanks for helping improve GitHub Mobile. 💖

**Reply by FWC-9696** [reply]
Thank you for the fast response! I appreciate it!

**Reply by FWC-9696** [reply]
I'm fairly certain you can take the link outside of the app, but that requires changing the "open by default" settings from the GitHub App page within Android settings. This _does_ work as a temporary workaround, though. Thanks again.

![Screenshot_20240411-112314_Settings.png](https://github.com/community/community/assets/76880537/418cb15a-d481-4045-866f-41c964d2e1cc)

**Reply by Phoenix616** [reply]
Would it be possible to just add an option to bypass the forms for now to add back the free form text field? (It's not like the forms are enforced on the server anyways) Might be faster than full form support...

Not being able to open issues on mobile is a pretty big blocker right now and having to enable that workaround and log in in the browser every time one wants to open an issue really slows the process down.

**Reply by TPS** [reply]
> Would it be possible to just add an option to bypass the forms for now to add back the free form text field? (It's not like the forms are enforced on the server anyways) Might be faster than full form support...

On projects where these templates are implemented, _bypassing_ them is a sure way to get your issue ignored or delayed.


> Not being able to open issues on mobile is a pretty big blocker right now and having to enable that workaround and log in in the browser every time one wants to open an issue really slows the process down.

This is utterly true.

**Comment by TPS** [comment]
Are https://github.com/orgs/community/discussions/15721 & https://github.com/orgs/community/discussions/108619 the same as this or just related?

**Reply by haIIux** [reply]
They appear to be the same, and likely can be merged together if possible.
